site/tests/integration/block-publish-npm.test.ts

```diff
@@ -60,15 +64,19 @@ test("user should be able to publish block & see it on the list", async ({
   ).toBeVisible();
 
   await expect(page.locator("[data-testid=list-view-card]")).toBeVisible();
-  await expect(page.locator("[data-testid=list-view-card] a")).toHaveText(
-    "@alice",
+  await expect(page.locator("[data-testid=list-view-card]")).toHaveText(
+    "test-npm-block block",
```

Comment:
Related to https://github.com/blockprotocol/blockprotocol/pull/642#discussion_r981597269